File: Project-Euler/Solutions-Simon/euler1/euler1.jl
#Find the sum of all the multiples of 3 or 5 below 1000.

function f1(n)
  sum = 0
  for i = 1:n-1
      if i%3 == 0 || i%5 == 0
        sum += i
      end
  end
  println(sum)
end
f1(1000)

File: Project-Euler/Solutions-Simon/euler1/euler1.java
public class euler1 {
	public static void main (String[] args)
	{
		System.out.println(f1(1000));
	}
	
	private static int f1 (int n)
	{
		int sum = 0;
		for (int i = 1; i < n; i++)
		{
			if (i % 3 == 0 || i % 5 == 0)
			{
				sum += i;
			} 
		}
		return sum;
	}
}
